pmtiles/intro.qmd

```diff
@@ -0,0 +1,97 @@
+---
+title: PMTiles
+subtitle: Guidelines for PMTiles
+---
+
+# PMTiles
+
+PMTiles is a single-file archive format for tiled data, usually used for visualization.
+
+PMTiles can be used for any format of tiled data. It's used most often with vector data, where each tile is encoded as a Mapbox Vector Tile (MVT), but can also be used with e.g. raster data or terrain mesh data.
+
+## Analytical vs tiled data formats
+
+To understand PMTiles, it's important to understand the difference between "analytical" data and "tiled" data. Analytical data refers to data in its original form, without any modifications to geometry. Tiled data formats apply a variety of modifications to geometries, including clipping and simplification, to save space and make it faster to visualize.
+
+![](../images/analytical_vs_tiled_geoms.png)
+
+Consider the above diagram. In an analytical format, every coordinate of the complex polygon would be included in the file. In a tiled format, the geometry would be split up into every tile it intersects, and possibly simplified as well.
```

Comment:
It's not clear what this means: "the geometry would be split up into every tile it intersects". If I was to guess you are saying "in a tiled format, there are predefined tile sets (or grids) and each tile set will have 1 or more tiles (cells of the grid) and the geometry would be split into tiles which it intersects"

I think it's the phrase "every tile" which is confusing me, makes it sounds like the whole geometry is in every tile it intersects